binr/r2pm/r2pm

```diff
@@ -243,7 +243,7 @@ R2PM_List() {
 R2PM_Search() {
 
 	#helper for pretty printing tabbed lists
-	if ! type "column" > /dev/null; then
+	if ! command -vp column > /dev/null; then
```

Comment:
i dont think type is a bashism. is it?

Reply:
https://wiki.ubuntu.com/DashAsBinSh#type

Before
```
 kij@rohana  ~/Project/radare2  ➦ 33a01a57f  checkbashisms binr/r2pm/r2pm 
possible bashism in binr/r2pm/r2pm line 246 (type):
	if ! type "column" > /dev/null; then
possible bashism in binr/r2pm/r2pm line 249 ($'...' should be "$(printf '...')"):
		cols() { column -s $'\t' -t; }
possible bashism in binr/r2pm/r2pm line 338 (unsafe echo with backslash):
	echo "\033[31mERROR: \033[32m$@\033[0m"
```
After:
```
 kij@rohana  ~/Project/radare2   master  checkbashisms binr/r2pm/r2pm
possible bashism in binr/r2pm/r2pm line 250 ($'...' should be "$(printf '...')"):
		cols() { column -s $'\t' -t; }
```

Reply:
i see som scary inconsistencies in the behaviour of `command -vp`

<img width="230" alt="screen shot 2018-08-23 at 01 26 00" src="https://user-images.githubusercontent.com/917142/44496165-83f32500-a673-11e8-868e-46ae9aef7437.png">

Reply:
That is expected due to the -p,  my mistake. -p should not be there.  I'll resubmit without  'column'